client/views/msg.tpl

```diff
@@ -1,6 +1,6 @@
 <div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}" id="msg-{{id}}" data-time="{{time}}" data-from="{{from}}">
-	<span class="time tooltipped tooltipped-e" aria-label="{{localetime time}}">
-		{{tz time}}
+	<span class="time tooltipped tooltipped-e {{#if seconds}}seconds{{/if}}" aria-label="{{localetime time}}">
```

Comment:
I think it's better to set option classes on `body`, like `option-seconds`. How would this mix with 12h+seconds display anyway? Perhaps it's better to make it a flexible column, if possible.